openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/1-serving-crds.yaml

```diff
@@ -590,9 +590,6 @@ spec:
                                 description: 'SecurityContext defines the security options the container should be run with. If set, the fields of SecurityContext override the equivalent fields of PodSecurityContext. More info: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/'
                                 type: object
                                 properties:
-                                  allowPrivilegeEscalation:
```

Comment:
This part is missing?

Reply:
Hmm... it seems we missed https://github.com/openshift-knative/serving/pull/18 for 1.7.